### ISSUE TITLE
another suggestion (actually useful)

### ISSUE THREAD

**Issue by ax3ro** [BlacketWare/v2#5]
another suggestion (this one will actually be useful) 

TokensAlert.js

a script that sends an alert every time you get tokens from chatting.
like how that used to be in the game, but was removed.

It can be in a way that the message alert doesn't have to be "modified".

like: alert ('You have got 5 tokens')

the script will run in the console for the whole time that you are in the "chat" page
the script goes away when you exit the chat page.

basically, the script checks how many tokens you have from the start, then sends an alert when the number of tokens goes up (no matter how many).

**Comment by Ljduley12** [comment]
Where
